charts/garden-extension-validator-gcp/charts/application/values.yaml

```diff
@@ -0,0 +1,8 @@
+global:
```

Comment:
This values.yaml should be a symlink to `../../values.yaml`

Reply:
packr seems to have an issue with symlinks?

Reply:
my bad forget about it.